Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -50,8 +50,8 @@
         variation: 0.125
     activatedDamage:
       types:
-        Slash: 15
-        Heat: 15
+        Slash: 25 # Delta V: Moved around damage values
```

Comment:
read CONTRIBUTING bruh

also these comments should just say `was (pevious value)`